homeassistant/components/wiz_light/light.py

```diff
@@ -0,0 +1,394 @@
+"""WiZ Light integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+from pywizlight import PilotBuilder, wizlight
+from pywizlight.bulblibrary import BulbType
+from pywizlight.exceptions import (
+    WizLightConnectionError,
+    WizLightNotKnownBulb,
+    WizLightTimeOutError,
+)
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    ATTR_COLOR_TEMP,
+    ATTR_EFFECT,
+    ATTR_HS_COLOR,
+    ATTR_RGB_COLOR,
+    SUPPORT_BRIGHTNESS,
+    SUPPORT_COLOR,
+    SUPPORT_COLOR_TEMP,
+    SUPPORT_EFFECT,
+    LightEntity,
+)
+from homeassistant.const import CONF_HOST, CONF_NAME
+import homeassistant.util.color as color_utils
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+SUPPORT_FEATURES_RGB = (
+    SUPPORT_BRIGHTNESS | SUPPORT_COLOR | SUPPORT_COLOR_TEMP | SUPPORT_EFFECT
+)
+SUPPORT_FEATURES_DIM = SUPPORT_BRIGHTNESS
+SUPPORT_FEATURES_WHITE = SUPPORT_BRIGHTNESS | SUPPORT_COLOR_TEMP
```

Comment:
These have been replaced by color modes (in theory they should have been removed already https://developers.home-assistant.io/docs/core/entity/light/#support-feature)

https://developers.home-assistant.io/docs/core/entity/light/#color-modes